packages/terra-i18n/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 4.37.0 - (February 16, 2021)
+
+* Changed
+  * Minor dependency version bump
+
```

Comment:
why is terra-i18n getting released again?

Reply:
@mjhenkes lerna did it?

Reply:
Good question though.